src/validations/role.validation.js

```diff
@@ -0,0 +1,43 @@
+import Joi from '@hapi/joi';
+import ResponseService from '../services/response.service';
+
+const userRoleSchema = Joi.object({
+  userEmail: Joi.string()
+    .email({ minDomainSegments: 2 })
+    .required()
+    .trim()
+    .messages({
+      'string.empty': 'Please enter userEmail',
+      'string.email': 'Enter valid email i.e: email@example.com',
+      'any.required': 'userEmail is required'
+    }),
+  userRole: Joi.string()
+    .trim()
+    .valid('travel_admin', 'travel_team_member', 'manager', 'requester')
+    .required()
+    .messages({
+      'string.base': 'Invalid type, userRole must be a string',
+      'string.empty': 'Please enter userRole',
+      'any.required': 'userRole is required'
+    })
+}).options({ abortEarly: false });
+
+const validateHandler = (schema, body, res, next) => {
+  const { error } = schema.validate(body);
+  if (error) {
+    const errors = [];
+    const { details } = error;
+    details.map(({ message }) => errors.push((message.replace(/[^a-zA-Z0-9 .-]/g, ''))));
```

Comment:
Refactor this line accordingly. This is the umpteenth time I will be giving feedback on how to use `.map`